lxd/loki/loki.go

```diff
@@ -330,11 +330,19 @@ func (c *Client) HandleEvent(event api.Event) {
 			}
 		}
 
+		keys := make([]string, len(context))
+
+		for k := range context {
+			keys = append(keys, k)
+		}
+
+		sort.Strings(keys)
+
 		messagePrefix := ""
```

Comment:
BTW when concatenating strings we've found it quicker to use strings Builder

https://pkg.go.dev/strings#Builder